src/Utilities/MakeWithValue.hpp

```diff
@@ -31,8 +32,8 @@ struct MakeWithValueImpl {
 /// at the same set of grid-points as the `input`
 ///
 /// \see MakeWithValueImpls
-template <typename R, typename T>
-SPECTRE_ALWAYS_INLINE R make_with_value(const T& input, double value) {
+template <typename R, typename T, typename ValType>
+SPECTRE_ALWAYS_INLINE R make_with_value(const T& input, ValType value) {
```

Comment:
Not your fault (our standards have changed since this code was written), but could you mark all the apply functions `noexcept`?